debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleClobDataTypeIT.java

```diff
@@ -1114,14 +1116,73 @@ public void shouldNotStreamAnyChangesWhenLobEraseIsDetected() throws Exception {
                 "amount := 10;" +
                 "dbms_lob.erase(loc_c, amount, 1); end;");
 
-        // Wait until the log has recorded the message.
         // Wait until the log has recorded the message.
         Awaitility.await().atMost(Duration.ofMinutes(1))
                 .until(() -> logminerLogInterceptor.containsWarnMessage("LOB_ERASE for table")
                         || xstreamLogInterceptor.containsWarnMessage("LOB_ERASE for table"));
         assertNoRecordsToConsume();
     }
 
+    @Test
+    @FixFor({ "DBZ-2948", "DBZ-5773" })
+    @SkipWhenAdapterNameIsNot(value = SkipWhenAdapterNameIsNot.AdapterName.OLR, reason = "OpenLogReplicator does not differentiate between LOB operations")
+    public void shouldStreamChangesWhenLobEraseIsDetected() throws Exception {
```

Comment:
Is this supported for XStream?

Reply:
This is the same behavior as from the Blob class, there are two tests with similar names, and the new test is to show that DBMS_LOB.ERASE cannot be differentiated with OpenLogReplicator and that we get the data in its _final_ form.